frontend/Makefile

```
@@ -0,0 +1,17 @@
+include ../mk/common.mk
+include ../mk/lint.mk
+include ../mk/test.mk
+
+.PHONY: all
+all: verifier
+
+verifier:
+	go build -o verifier cmd/veraison/main.go
```

without an explicit dependency on the source, the rule becomes quickly inert
```suggestion
verifier: cmd/veraison/main.go
	go build -o $@ $<
```